central/scannerdefinitions/handler/handler.go

```diff
@@ -421,8 +444,10 @@ func (h *httpHandler) openMostRecentDefinitions(ctx context.Context, uuid string
 	return
 }
 
-func (h *httpHandler) openMostRecentV4File(t updaterType, updaterKey, fileName string) (file *vulDefFile, err error) {
-	// TODO(ROX-20520): enable fetching offline file
+func (h *httpHandler) openMostRecentV4File(ctx context.Context, t updaterType, updaterKey, fileName string) (file *vulDefFile, err error) {
```

Comment:
Similar to `openMostRecentDefinitions` we need to choose the correct file between an uploaded "offline" bundle and an "online" one. This is to allow for the case where online updates are not working due to some firewall error or some other reason, yet a user is able to upload a vuln feed. We need to look at both and choose the most recent one